### ISSUE TITLE
Bad example sentence for the entry <ⲟⲩⲱϣⲉ>

### ISSUE THREAD

**Issue by mark-yacoub** [KELLIA/dictionary#215]
Word you searched for in the dictionary (in Coptic):

ⲟⲩⲱϣⲉ

TLA ID (shown at the top of the entry page after "TLA Lemma No."): 

C5268

Bad example sentence (insert your example):

crum shows ⲟⲩⲱϣⲉ as a variance of ⲟⲩⲱϣ
The dictionary uses only "consume" as the meaning
https://coptot.manuscriptroom.com/crum-coptic-dictionary/?docID=800000&pageID=500a&tla=C5257

**Comment by amir-zeldes** [comment]
Yes, thanks for reporting - I agree it should be a legitimate alternate form for ⲟⲩⲱϣ as well. However the current CDO lemmas generally don't distinguish all inflected or alternative forms of verbs (for example, the entry for a verb like ⲥⲱⲧⲡ does not list forms such as ⲥⲟⲧⲡ and ⲥⲉⲧⲡ). We do have redirect links for some of these if they are attested with that lemma in Coptic Scriptorium data, so if you search for ⲥⲉⲧⲡ you will be offered a redirect:

https://coptic-dictionary.org/results.cgi?quick_search=%E2%B2%A5%E2%B2%89%E2%B2%A7%E2%B2%A1

ⲟⲩⲱϣⲉ does not occur in our data, so it doesn't happen there, and in any case it prefers matching an actual entry, so we would get to C5268. @dwerning / @simondschweitzer - unusual forms like ⲟⲩⲱϣⲉ are not strictly predictable like alternations of the ⲥⲟⲧⲡ/ⲥⲉⲧⲡ kind - is there a plan to add such forms to entries like C5257 and C5258? I can't add it on my end without a TLA form ID being minted.
